packages/runtime/container-runtime/src/summary/summaryManager.ts

```diff
@@ -186,15 +190,11 @@ export class SummaryManager extends TypedEventEmitter<ISummarizerEvents> impleme
 			return { shouldSummarize: false, stopReason: "notElectedClient" };
 		}
 
-		if (!this.connectedState.connected) {
+		if (!this.connectedState.connected || this.disposed) {
```

Comment:
this.disposed has already been checked above.

Reply:
Oops thanks